data/image_train_item.py

```diff
@@ -232,15 +235,15 @@ def hydrate(self, crop=False, save=False, crop_jitter=20):
         # print(self.image.shape)
 
         return self
-    
+
     def __compute_target_width_height(self):
         self.target_wh = None
         try:
-            with self.load_image() as image:
+            with self.load_image(convert_rgb=False) as image:
```

Comment:
The conver_rgb arg seems unnecessary here, can just open the image here with `with PIL.Image.open(self.pathname) as image:` since no transform is needed.

Reply:
partly it is adding unnecessary arg to control a switch elsewhere when a one-liner open will work, and the convert_rgb is actually doing two things, converting but also doing exif transposing, so its not really accurate.  Also removes the extra if statement in load image.